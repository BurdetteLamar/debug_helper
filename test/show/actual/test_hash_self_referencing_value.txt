---
Hash (size=1 message='test_hash_self_referencing_value'):
  Pair 0:
    Key:
      Symbol (size=1):
        to_s: a
        encoding: !ruby/encoding US-ASCII
    Value: Hash {:a=>{...}}
