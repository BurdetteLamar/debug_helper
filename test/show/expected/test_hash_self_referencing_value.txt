---
Hash (message='test_hash_self_referencing_value'):
  size: 1
  default: 
  default_proc: 
  Pair 0:
    Key:
      Symbol:
        to_s: a
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value: Hash {:a=>{...}}
